server/game/game.js

```diff
@@ -57,6 +58,10 @@ class Game extends EventEmitter {
         return player.constructor === Spectator;
     }
 
+    hasActivePlayer(playerName) {
+        return !!this.players[playerName] && !this.players[playerName].left;
```

Comment:
Checking for truthiness is fine here, rather than forcing it to bool, but I'm happy to leave it as is too